src/pages/ExternalWorkshops/ExternalWorkshops.module.css

```diff
@@ -0,0 +1,51 @@
+.ExternalWorkshops{
+    font-size: 1.3rem;
+    color: #484848;
+}
+
+.TitleBar{
+    background-color: #002A56;
+    color: white;
+    /* padding: 24px 86px; */
+    padding: 24px 6%;
+
+}
+
+.TitleBar h1{
+    font-weight: 600;
+    font-size: 54px;
+}
+
+.NavLinks{
+    display: flex;
+    text-align: left;
+    font-size: 0.75em;
+    padding-top: 12px;
+    padding-bottom: 20px;
```

Comment:
Styling generally looks good to me. Perhaps you can consider vh for the vertical relative unit, vw forhorizontal relative unit,  to improve responsiveness.